usmqe/ceph/ceph_cluster.py

```diff
@@ -190,9 +190,7 @@ def fsid(self):
 
             {'fsid': 'aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee'}
         """
-        # WORKAROUND for https://github.com/tomerfiliba/plumbum/issues/275
-        # return self.run_on_mon('fsid')
-        return self.run_on_mon('fsid && echo || (echo; false)')
+        return self.run_on_mon('fsid')
```

Comment:
Does this mean that the workaround never actually worked? I tried to check commit message and comments, but don't see the answer there. @dahorak @mkudlej ?